sources/Core/Numerics/Quaternion.cs

```diff
@@ -21,74 +23,165 @@ namespace TerraFX.Numerics
-                && (left.W == right.W);
-        }
+        public static bool operator ==(Quaternion left, Quaternion right) => left._value == right._value;
 
         /// <summary>Compares two <see cref="Quaternion" /> instances to determine inequality.</summary>
         /// <param name="left">The <see cref="Quaternion" /> to compare with <paramref name="right" />.</param>
         /// <param name="right">The <see cref="Quaternion" /> to compare with <paramref name="left" />.</param>
         /// <returns><c>true</c> if <paramref name="left" /> and <paramref name="right" /> are not equal; otherwise, <c>false</c>.</returns>
-        public static bool operator !=(Quaternion left, Quaternion right)
+        public static bool operator !=(Quaternion left, Quaternion right) => left._value != right._value;
+
+        /// <summary>Returns the value of the <see cref="Quaternion" /> operand (the sign of the operand is unchanged).</summary>
+        /// <param name="value">The operand to return</param>
+        /// <returns>The value of the operand, <paramref name="value" />.</returns>
+        public static Quaternion operator +(Quaternion value) => value;
+
+        /// <summary>Negates the value of the specified <see cref="Quaternion" /> operand.</summary>
+        /// <param name="value">The value to negate.</param>
+        /// <returns>The result of <paramref name="value" /> multiplied by negative one (-1).</returns>
+        public static Quaternion operator -(Quaternion value) => value * -1;
+
+        /// <summary>Adds two specified <see cref="Quaternion" /> values.</summary>
+        /// <param name="left">The first value to add.</param>
+        /// <param name="right">The second value to add.</param>
+        /// <returns>The result of adding <paramref name="left" /> and <paramref name="right" />.</returns>
+        public static Quaternion operator +(Quaternion left, Quaternion right) => new Quaternion(left._value + right._value);
+
+        /// <summary>Subtracts two specified <see cref="Quaternion" /> values.</summary>
+        /// <param name="left">The minuend.</param>
+        /// <param name="right">The subtrahend.</param>
+        /// <returns>The result of subtracting <paramref name="right" /> from <paramref name="left" />.</returns>
+        public static Quaternion operator -(Quaternion left, Quaternion right) => new Quaternion(left._value - right._value);
+
+        /// <summary>Multiplies two specified <see cref="Quaternion" /> values. Is the same as <seealso cref="Quaternion.Concatenate" /> with reversed operand order.</summary>
+        /// <param name="left">The first value to multiply.</param>
+        /// <param name="right">The second value to multiply.</param>
+        /// <returns>The result of multiplying <paramref name="left" /> by <paramref name="right" />.</returns>
+        public static Quaternion operator *(Quaternion left, Quaternion right) => Quaternion.Concatenate(right, left);
```

Comment:
This is still not quite right. The code you have under `Concatenate` needs to be under `multiply` and `Concatenate needs to be `=> (right * left)`.

If you went with the approach where `operator *` forwards to `Concatenate`, then the algorithm under `Concatenate` needs to be fixed (all usages of left/right swapped). However, `Concatenate` is really a special application of multiplication rather than the other way around, so I'd rather the actual algorithm code just be here under `operator *`.

```suggestion
        public static Quaternion operator *(Quaternion left, Quaternion right)
        {
            return new Quaternion(
                Quaternion.Dot(left, new Quaternion(+right.W, +right.Z, -right.Y, +right.X)),
                Quaternion.Dot(left, new Quaternion(-right.Z, +right.W, +right.X, +right.Y)),
                Quaternion.Dot(left, new Quaternion(+right.Y, -right.X, +right.W, +right.Z)),
                Quaternion.Dot(left, new Quaternion(-right.X, -right.Y, -right.Z, +right.W))
            );
        }
```

Reply:
ok. I hope I got it correct now ;)